File: app/src/main/java/com/bresiu/codechallenge/repository/mappers/AlbumModelToEntityMapper.java
package com.bresiu.codechallenge.repository.mappers;

import com.bresiu.codechallenge.data.entity.Album;

public class AlbumModelToEntityMapper
		implements Mapper.ObjectMapper<Album, com.bresiu.codechallenge.model.Album> {

	@Override public Album map(com.bresiu.codechallenge.model.Album modelAlbum) {
		return new Album(modelAlbum.id, modelAlbum.userId, modelAlbum.title);
	}
}

File: app/src/main/java/com/bresiu/codechallenge/repository/mappers/AlbumModelToEntityMapper.kt
package com.bresiu.codechallenge.repository.mappers

import com.bresiu.codechallenge.data.entity.Album

class AlbumModelToEntityMapper : Mapper.ObjectMapper<Album, com.bresiu.codechallenge.model.Album> {

  override fun map(mappingItem: com.bresiu.codechallenge.model.Album): Album {
    return Album(mappingItem.id, mappingItem.userId, mappingItem.title)
  }
}
